targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/api/analogin_api.c

```diff
@@ -38,6 +38,14 @@ void analogin_init(analogin_t *obj, PinName pin)
 
     uint32_t instance = obj->adc >> ADC_INSTANCE_SHIFT;
     adc_config_t adc_config;
+    uint32_t reg;
+    uint32_t pin_number = pin & 0x1F;
+    uint8_t port_number = pin / 32;
+
+    /* Clear the PDEN_ADC0 bit in the PDRUNCFG0 */
```

Comment:
This bit is cleared inside  ADC_ClockPower_Configuration() function. The below lines of code can be deleted.

Reply:
My debugger is saying its not cleared and i get wrong measurements, always around 1700.

Reply:
* If i delete lines:
![withoutlines](https://user-images.githubusercontent.com/25270775/32959041-9c1c5e6e-cbc0-11e7-8a02-2470e2d80f89.PNG)

* If i dont:
![withlines](https://user-images.githubusercontent.com/25270775/32959046-a1718b96-cbc0-11e7-9516-af5ef2139b0a.PNG)

Reply:
Its called DIGIMODE bit. Ill correct